src/main/java/com/company/managementservice/model/entity/Salary.java

```diff
@@ -0,0 +1,38 @@
+package com.company.managementservice.model.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "salary")
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Salary extends AbstractEntity<Long>{
+
+    @JoinColumn(name = "employee_id", referencedColumnName = "id")
```

Comment:
name = salary_id, referencedColumnName = id. We are in salary entity and referring to employee